utils/urls.js

```diff
@@ -0,0 +1,3 @@
+const { PREDICTION_URL, CURRENT_USER } = require('sharify').data
+
+export const liveAuctionUrl = (id) => `${PREDICTION_URL}/${id}${CURRENT_USER ? '/login' : ''}`
```

Comment:
Yup, lets stick to our department :) We shouldn't be worrying about global concerns just yet, and instead opt in to refactoring those out later (if need be) all at once.